playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -57,6 +57,7 @@
 
   - name: Add secrets to registry service account
     command: oc secrets add serviceaccount/registry secrets/dockerregistry
+    run_once: true
```

Comment:
If I'm reading this correctly, all of this really just needs to be done once and not just this single step.

It might be better to have a separate play at the beginning of this file to create a new group that contains just a single master host.  Something like this should work:
```
hosts: localhost
become: no
sudo: no
tasks:
- add_host:
    name: hostvars[item].name
    group: oo_masters
  with_items: groups['tag_clusterid_' ~ cluster_id] | default([]) | intersect(groups['tag_host-type_master'] | default([]))
```

And then for this play, you could just specify:
```
hosts: oo_masters[0]
```